packages/manager/apps/pci-object-storage/src/api/hooks/useContainer.ts

```diff
@@ -40,6 +40,7 @@ export const useServerContainer = (
     }),
     queryFn: () => getServerContainer(projectId, region, name, id),
     enabled: !!projectId && !!name && !!region,
+    staleTime: 0,
```

Comment:
staleTime 0 will force the ui to refetch the data each time it is requested, any reason for that ? is it not possible to invalidate the cache manually if you need to have fresh data ?